applications/GeoMechanicsApplication/tests/shell_with_reset_displacement/ProjectParameters.json

```diff
@@ -160,8 +136,8 @@
         "Parameters":    {
             "model_part_name": "PorousDomain.ZsupportLoadPoint",
             "variable_name":   "DISPLACEMENT",
-            "active":          [false,false,true],
-            "is_fixed":        [false,false,true],
+            "active":          [true,true,true],
```

Comment:
This is for @WPK4FEM, maybe I'm not understanding this correctly but shouldn't these be false for X and Y?